app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.all
+    # render json: @greetings, status: :ok
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.